doc/man3/RAND_DRBG_reseed.pod

```diff
@@ -0,0 +1,84 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_reseed,
+RAND_DRBG_set_reseed_interval,
+RAND_DRBG_set_reseed_time_interval
+- reseed the RAND_DRBG instance
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ int RAND_DRBG_reseed(RAND_DRBG *drbg,
+                      const unsigned char *adin, size_t adinlen);
+
+ int RAND_DRBG_set_reseed_interval(RAND_DRBG *drbg,
+                                   unsigned int interval);
+
+ int RAND_DRBG_set_reseed_time_interval(RAND_DRBG *drbg,
+                                        time_t interval);
+
+
+=head1 DESCRIPTION
+
+These functions are part of the OpenSSL L<RAND_DRBG(7)> API.
+
+RAND_DRBG_reseed()
+reseeds the given B<drbg>,  mixing in the specified additional data provided
+in the buffer B<adin> of length B<adinlen>.
+The additional data can be omitted by setting B<adin>=NULL and B<adinlen>=0.
```

Comment:
Maybe it should mention something about that it gets it's data either from the parent drbg or it's default source?

Reply:
Or maybe it's already clear to people what reseed does

Reply:
Done.